### ISSUE TITLE
I'm curious about the number of images and epochs used in training.

### ISSUE THREAD

**Issue by greentec** [yjn870/REDNet-pytorch#2]
Hello.
I used the BSD dataset as stated in the paper, and your code for training. I trained for 20 epochs using all of train data, validation data, and test data, but the final loss was around 0.002 ~ 0.003. This is not enough for PSNR to be 30, which is not the quality of the sample output you uploaded.
Can you tell me your learning environment? Thanks in advance.

**Comment by circlehy** [comment]
> Hello.
> I used the BSD dataset as stated in the paper, and your code for training. I trained for 20 epochs using all of train data, validation data, and test data, but the final loss was around 0.002 ~ 0.003. This is not enough for PSNR to be 30, which is not the quality of the sample output you uploaded.
> Can you tell me your learning environment? Thanks in advance.

I have trained the rednet30 using bsd400 for 100 epoch with AWGN sigma=10, which the training figure in the paper did,  tested on the same dataset the psnr is around 31, much lower than the results in the paper. The whole paper did not provide the training details like training epochs and batch size and so on.

**Comment by greentec** [comment]
Thank you for the details of the training missing from the paper.
